### ISSUE TITLE
Unable to run Appium with app capability set to URL+.app name for iOS

### ISSUE THREAD

**Issue by SaurabhDeshpande-OpenX** [appium/appium#10890]
## The problem

As per Appium docs one can specify a URL for iOS App file. The App file will get downloaded and installed on the iOS Simulator.
However, this does not work as expected i get below error:

org.openqa.selenium.WebDriverException: An unknown server-side error occurred while processing the command. Original error: Could not extract Info.plist from application: Plist file doesn't exist: '/var/folders/bv/rwv42gxs2ds_m22yqyz6406s0r0mln/T/2018515-32043-tqrcal.p5bc/appium-app.app/Info.plist' (WARNING: The server did not provide any stacktrace information)

The same App file when specified as a File Path in Mac runs successfully.

## Environment

* Appium version (or git revision) that exhibits the issue: 1.8.1
* Last Appium version that did not exhibit the issue (if applicable):
* Desktop OS/version used to run Appium: Mac High Sierra +Xcode 9.4
* Node.js version (unless using Appium.app|exe): 9.11.1
* Mobile platform/version under test: iPhone Simulator 11.2
* Real device or emulator/simulator: Simulator
* Appium CLI or Appium.app|exe: Appium CLI

## Details

If necessary, describe the problem you have been experiencing in more detail.

## Link to Appium logs

Create a [GIST](https://gist.github.com) which is a paste of your _full_ Appium logs, and link them here. 
https://gist.github.com/SaurabhDeshpande-OpenX/8414b589290a19ce81fa784e2b53c41f
Do _NOT_ paste your full Appium logs here, as it will make this issue very long and hard to read! 
If you are reporting a bug, _always_ include Appium logs!

## Code To Reproduce Issue [ Good To Have ]

Please remember that with sample code it's easier to reproduce the bug and it's much faster to fix it.

Please git clone https://github.com/appium/appium and from the `sample-code` directory, use one of your favourite languages and sample apps to reproduce the issue.

In case a similar scenario is missing in sample-code, please submit a PR with one of the sample apps provided.

**Comment by mykola-mokhnach** [comment]
Yes, the application should either be zipped or packed into .ipa. One cannot download directory structures.
